code/modules/projectiles/guns/ds13/contactbeam.dm

```diff
@@ -17,6 +17,7 @@
 	icon_state = "contact"
 	item_state = "contact"
 	wielded_item_state = "contact-wielded"
+	w_class = ITEM_SIZE_LARGE
```

Comment:
long guns are meant to be huge sized so they don't fit in backpacks or rig storage, special methods of carrying them will be added in future

Reply:
Oversight, will edit.